ui/src/app/tasks-jobs/tasks/tasks.component.html

```diff
@@ -1,11 +1,15 @@
 <h1 class="page-title-actions">
-  Tasks
+  {{ 'tasks.main.title' | translate }}
   <span class="actions">
     <button type="button" class="btn btn-sm btn-primary" (click)="createTask()" [appRole]="['ROLE_CREATE']">
-      Create task
+      {{ 'tasks.main.create' | translate }}
+    </button>
+    <button type="button" class="btn btn-sm btn-secondary" grafanaDashboardTasks>
+      {{ 'tasks.main.grafana' | translate }}
```

Comment:
Texts like `Grafana Dashboard` and `Wavefront Dashboard` are visible on other pages.
Can we move these to `commons` translations ?